google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineEnvironment.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2016 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.appengine.cloud;
+
+import com.google.cloud.tools.intellij.util.GctBundle;
+
+/**
+ * Specifies an App Engine environment.
+ */
+public enum AppEngineEnvironment {
+  APP_ENGINE_STANDARD("appengine.environment.name.standard"),
+  APP_ENGINE_FLEX("appengine.environment.name.flexible");
+
+  private String label;
+
+  AppEngineEnvironment(String label) {
+    this.label = label;
+  }
+
+  @Override
+  public String toString() {
```

Comment:
Really? With toString its the string representation of an AppEngineEnvironment which seemed pretty clear to me. It also has the benefit of not having to call a method for things like logging. I don't have strong feelings about this though.